README.md

```diff
@@ -7,30 +7,149 @@
 [![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-mongodb/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-mongodb/actions/workflows/build-with-bal-test-native.yml)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+## Overview
+
 [MongoDB](https://docs.mongodb.com/v4.2/) is a general purpose, document-based, distributed database built for modern application developers and for the cloud era. MongoDB offers both a Community and an Enterprise version of the database.
 
-MongoDB connector connects to MongoDB from [Ballerina](https://ballerina.io/). This connector provides the capability to perform the MongoDB CRUD operations.
+The `ballerinax/mongodb` package offers APIs to connect to MongoDB servers and to perform various operations including CRUD operations, indexing, and aggregation. 
+
+The Ballerina MongoDB connector uses the MongoDB Java Sync library underneath. It is compatible with MongoDB 3.6 and later versions.
```

Comment:
Here do we need to mention about our native dependencies? Just mentioning the compatible mongoDB version(or the range) wouldn't be sufficient?

```suggestion
The Ballerina MongoDB connector uses the MongoDB Java Sync library underneath. It is compatible with MongoDB 3.6 and later versions.
```

Reply:
+1

Reply:
Fixed with 3c16d95b6a552b34344ee408ecd6d3eaace659c5